calico-cloud/threat/container-threat-detection.mdx

```diff
@@ -37,6 +37,9 @@ Our threat detection engine also monitors activity within the containers running
 
 ## Before you begin...
 
+**Not supported:**
```

Comment:
This needs to go in calico-cloud_versioned_docs/version-3.17/threat/container-threat-detection.mdx as well.